### PR TITLE
Added GCC 13 support

### PR DESCRIPTION
**Pull request by xvitaly**
Added GCC 13 support.

### CODE REVIEW

**Review by SSE4 — APPROVED** [review]

**Comment by H-G-Hristov** [comment]
@memsharded Will this be included in the 1.x.x? GCC 13.1 has been released: https://gcc.gnu.org

I need GCC 13/14... support in the 1.x.x for some time as I won't be able to move to conan 2 any time soon.

File: conans/client/conf/__init__.py [hunk 1]
```diff
@@ -94,7 +94,8 @@
                       "9", "9.1", "9.2", "9.3", "9.4", "9.5",
                       "10", "10.1", "10.2", "10.3", "10.4",
                       "11", "11.1", "11.2", "11.3",
-                      "12", "12.1", "12.2"]
+                      "12", "12.1", "12.2",
+                      "13"]
```
Comment (H-G-Hristov) [inline conans/client/conf/__init__.py:98]:
```suggestion
                      "13", "13.1"]
```

**Review by H-G-Hristov — CHANGES_REQUESTED** [review]

**Comment by memsharded** [comment]
Yes, lets add this to default settings, sorry for the wait, it is being super busy here with 2.0

**Comment by memsharded** [comment]
https://github.com/conan-io/conan/pull/13848 for next 1.60 (and also 2.0.5)

**Pull request closed without merging**